tests/sparsezoo/utils/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
let's not include these changes right now, since they're not in the scope for testing `iter_batches`

Reply:
Will have to move the tests/utils.py file, if we need to keep test_data.py in tests/utils/test_data.py
Can keep it as is if we can change the tests/utils folder to something else.

Reply:
good catch.  Can we rename `tests/utils.py` to `tests/helpers.py`? Will need to make sure to rename all instances of importing `test.utils` from tests to `test.helpers` (and rerun all sparsezoo tests to be safe)

Reply:
Resolved!